examples/javascript/index.js

```diff
@@ -0,0 +1,140 @@
+var math = require('mathjs');
+
+data1 = [
+  [0, 10, 10, 80, 0],
+  [10, 0, 10, 0, 80],
+  [10, 10, 0, 40, 40],
+  [0, 0, 0, 1, 0],
+  [0, 0, 0, 0, 1],
+];
+
+data2 = [
+    [  1,   0,   0,   0,   0],
+    [0.5,   0, 0.5,   0,   0],
+    [  0, 0.5,   0, 0.5,   0],
+    [  0,   0, 0.5,   0, 0.5],
+    [  0,   0,   0,   0,   1],
+];
+
+function range(size, start_at= 0) {
```

Comment:
what is "start_at" (why would it change)?

Reply:
Python has a `range()` function in the standard library that generates the numbers in a range: 
```python
>>> range(5)
[0, 1, 2, 3, 4]
```

One way to call it is with a start_at:
```python
>>> range(10, 15)
[10, 11, 12, 13, 14]
```

Since javascript didn't have it, I wrote it in!

Here, when we split the matrix into 2 smaller matrices, we need 2 ranges.
One from `1-n` and one from `n-k`.

Reply:
You can see it used [here](https://github.com/liquid-donations/reimagine-giving/pull/15/files#diff-b736672b38345713af819ef43f3ff4a095a8ac9b2421e20aa04d23fa39cccdf6R26)

Reply:
The syntax of the javascript function is a little different `start_at`  than the python example I gave.
If you wanted it to generate an array of numbers 10-15, you'd say:
```javascript
range(15, 10)
```
"Range to 15, start at 10."